sample/package.json

```diff
@@ -43,13 +43,14 @@
     "ms-vscode.azure-account"
   ],
   "devDependencies": {
-    "@types/node": "^12.0.0",
-    "@types/vscode": "1.16.0",
+    "@types/node": "^14.0.0",
+    "@types/vscode": "^1.53.0",
     "typescript": "^4.0.2"
   },
   "dependencies": {
     "@azure/arm-appservice": "^9.0.0",
     "@azure/arm-resources": "^4.2.2",
-    "@azure/arm-subscriptions": "^3.1.2"
+    "@azure/arm-subscriptions": "^3.1.2",
+    "@microsoft/vscode-azext-utils": "^0.1.0"
```

Comment:
Is it possible for extensions to use the API without taking a dependency on this package?

Reply:
We need some sort of dependency on `@microsoft/vscode-azext-utils` so that `getApi` can be used to access a versioned API. Otherwise people would fall back to using the legacy API which I'd like to avoid.

The Account Extension has a hard dependency on `@microsoft/vscode-azext-utils` so we can specify it as a dev dependency in the sample and it will have access to that package at runtime.